features/product/viewing_product_in_admin_panel/viewing_details_of_simple_product.feature

```diff
@@ -0,0 +1,84 @@
+@viewing_products
+Feature: Viewing details of a simple product
+    In order to view detailed product information
+    As an Administrator
+    I want to be able to view product show page
+
+    Background:
+        Given the store operates on a single channel in "United States"
```

Comment:
This background is hardly readable, could we move the steps that are used in only one or two scenarios to those scenarios `Given` steps? Eg. associations are used only in one scenario but include three steps in the background.

Reply:
The other interesting conversation could be started whether Behat and BDD are a good solution to test a page that shows data.